### DISCUSSION TITLE
Devices are untagged

### DISCUSSION THREAD

**Discussion by kpnobvious** [jertel/vuegraf#17] [Q&A] [answered]
Hi,
I just started using this mid update with the API changes.  I am pulling in data now, so thanks for fixing that quickly.

I initially did not have my device names listed out in the vuegraf.json file and added them later and restarted the process.  In grafana, they are listed as "<account_name>-<circuit #>".  Vuegraf does list the channel names correctly.

In the influxdb, if I do a show series, they also have the same name format as in grafana.

Do i have to reset the DB because i updated the json afterwards?

influxdb:
> show series
key
---
energy_usage,account_name=Home,device_name=Home-1
...

log from vuegraf:
2021-02-10 18:52:21.228042 | INFO  | Login completed
2021-02-10 18:52:22.470379 | INFO  | Discovered new channel: Home (1,2,3) aka Mains A/B/C
2021-02-10 18:52:23.006676 | INFO  | Discovered new channel: Garage (1) (aka device in circuit 1)

**Comment by jertel** [comment, marked as answer]
It sounds like the device names in the JSON don't match what's coming back from the Emporia API. If they match correctly then vuegraf will use the channel names in your JSON file, regardless of what you originally had injected into the InfluxDB.

That said, you will want to reset your Influx DB anyway to clear out the badly named data.

**Reply by kpnobvious** [reply]
Thanks it was bad JSON.  Once i fixed that it straightened itself out.  It still looked funny in Grafana, but i realized that was due to the query time frame.

Now i just have to get it to run as a service and log and i'll be all set.  This very easy to use, thank you!

**Reply by kpnobvious** [reply]
As a bit of an aside, is there a reason you're pulling data every second vs longer intervals?  I wasn't sure if that was a limitation in the API, but i think it allows for different query intervals and just updating say every 15 minutes for that average.

**Reply by jertel** [reply]
It's not pulling data every second. It queries the Emporia API server every minute and asks for the data points for each second of that previous minute. I do not want to wait 15 minutes because I have alarms set for sensitive equipment that needs more timely response in the event of a power problem. I could have it only give me one data point per minute but I would miss out on rapid power spikes that can be indicative of certain equipment about to fail.
